portfolio/src/main/java/com/google/sps/data/FindSchedule.java

```diff
@@ -0,0 +1,116 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+
+public final class FindSchedule {
+  private static final Comparator<CalendarEvent> sortAscending =
+      Comparator.comparingLong(CalendarEvent::getStartTimeLong);
+
+  private static final Comparator<Task> sortByDuration =
+      Comparator.comparing(Task::getDurationSeconds).thenComparing(Task::getName);
+
+  /**
+   * This method returns a Task Collection which the greedy algorithm
+   * determines should be scheduled for the person.
+   */
+  public Collection<Task> greedy(Collection<CalendarEvent> events, Collection<Task> tasks,
+      long workHoursStartTimeSeconds, long workHoursEndTimeSeconds) {
+    List<CalendarEvent> eventsList = new ArrayList<CalendarEvent>(events);
+    Task[] tasksArray = tasks.toArray(new Task[tasks.size()]);
```

Comment:
As I wrote in my comment below, consider using list instead of array, unless there is a strong reason to use array.

Reply:
Done.